README.md

```diff
@@ -29,7 +29,9 @@ The **SponsorFaucet** returns a **rawTx** with suggested gas and input data.
    /**
     * @param dapp The address of dapp 
     */
-   async isAppliable(dapp) -> bool
+   async isAppliable(dapp) -> 
+   bool true // success
+   string //error message
```

Comment:
why not return null or empty string if appliable?

furthermore, could rename the API to checkAppliable or validateAppliable?

Reply:
modified return and function name